File: app/src/main/java/com/mrgames13/jimdo/feinstaubapp/CommonObjects/ExternalSensor.java
package com.mrgames13.jimdo.feinstaubapp.CommonObjects;

public class ExternalSensor {

    //Konstanten

    //Variablen als Objekte

    //Variablen
    private String chip_id = "no_id";
    private double lat = 0;
    private double lng = 0;

    public ExternalSensor() {}

    public ExternalSensor(String chip_id, double lat, double lng) {
        this.chip_id = chip_id;
        this.lat = lat;
        this.lng = lng;
    }

    public String getChipID() {
        return chip_id;
    }
    public void setChipID(String chip_id) {
        this.chip_id = chip_id;
    }

    public double getLat() {
        return lat;
    }
    public void setLat(double lat) {
        this.lat = lat;
    }

    public double getLng() {
        return lng;
    }
    public void setLng(double lng) {
        this.lng = lng;
    }
}

File: app/src/main/java/com/mrgames13/jimdo/feinstaubapp/CommonObjects/ExternalSensor.kt
/*
 * Copyright © 2019 Marc Auberer. All rights reserved.
 */

package com.mrgames13.jimdo.feinstaubapp.CommonObjects

class ExternalSensor {

    // Variables
    var chipID = "no_id"
    var lat = 0.0
    var lng = 0.0

    constructor()

    constructor(chip_id: String, lat: Double, lng: Double) {
        this.chipID = chip_id
        this.lat = lat
        this.lng = lng
    }
}
